src/side-loaded/basic-sideload/verifier-zkapp.ts

```diff
@@ -0,0 +1,110 @@
+import {
+  SmartContract,
+  state,
+  State,
+  method,
+  Field,
+  PublicKey,
+  VerificationKey,
+  DeployArgs,
+  Permissions,
+} from 'o1js';
+
+import {
+  EcdsaSideloadedProof,
+  KeccakSideloadedProof,
+} from './sideloaded-zkprograms.js';
+
+export { MyFavoriteZkApp };
+
+class MyFavoriteZkApp extends SmartContract {
```

Comment:
Nit: very minor, but could we rename this contract to something a little more descriptive just because this is serving as an example. I think it would slightly reduce the cognitive overhead for someone going through this code for the first time.